mix.exs

```diff
@@ -27,8 +27,8 @@ defmodule KinoExplorer.MixProject do
 
   defp deps do
     [
-      {:kino, "~> 0.8"},
-      {:explorer, "~> 0.5.0-dev", github: "elixir-nx/explorer"},
+      {:kino, "~> 0.8", github: "livebook-dev/kino"},
+      {:explorer, github: "elixir-nx/explorer", branch: "main"},
```

Comment:
Now that we only use functions in the Series module, I think we can use a released version of Explorer, no?

Reply:
```suggestion
      {:explorer, "~> 0.4.0"},
```

Reply:
Nevermind, we need nil_count still.